web-utils/build.gradle.kts

```diff
@@ -122,8 +124,10 @@ publishing {
 
         signing {
             val signingKeyId: String? = System.getenv("SIGNING_KEY_ID")
-            val signingKey: String? = System.getenv("SIGNING_KEY")
             val signingPassword: String? = System.getenv("SIGNING_PASSWORD")
+            val signingKey: String? = System.getenv("SIGNING_KEY")?.let { base64Key ->
+                Base64.getMimeDecoder().decode(base64Key).map{ it.toChar()}.joinToString("")
```

Comment:
i think we should not walk by each byte.
```
val decodedBytes = Base64.getDecoder().decode(encodedString)
val decodedString = String(decodedBytes)
```
here sample with creation string from bytes